onnx/defs/schema.cc

```diff
@@ -990,7 +990,8 @@ const std::vector<std::string>& OpSchema::all_numeric_types_ir12() {
       "tensor(int8)",         "tensor(int16)",          "tensor(int32)",      "tensor(int64)",
       "tensor(float16)",      "tensor(float)",          "tensor(double)",     "tensor(bfloat16)",
       "tensor(float8e4m3fn)", "tensor(float8e4m3fnuz)", "tensor(float8e5m2)", "tensor(float8e5m2fnuz)",
-      "tensor(uint4)",        "tensor(int4)",           "tensor(float4e2m1)", "tensor(float8e8m0)"};
+      "tensor(uint4)",        "tensor(int4)",           "tensor(float4e2m1)", "tensor(float8e8m0)",
```

Comment:
Will QuantizeLinear and DequantizeLinear be updated?
I think ConstantOfShape should also be updated?